developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -10,25 +10,55 @@ Foreman will run with the following requirements (aside from rubygem dependencie
 * Ruby 2.7.x
 * NodeJS 14
 * NPM 6.x
-* PostgreSQL 13
+* PostgreSQL 10+ (12+ recommended)
 
-It’s recommended to install https://github.com/rbenv/rbenv[rbenv] or https://github.com/rvm/rvm[rvm] to manage ruby, and https://github.com/nvm-sh/nvm[nvm] to manage node and npm on your system.
-* For Fedora users, you can use this https://developer.fedoraproject.org/start/sw/web-app/rails.html[guide] for installing: rbenv, Ruby & Bundler.
-(There is no need to install Rails, as it is installed by Bundler when running `bundle install` from Foreman directory).
+https://github.com/rbenv/rbenv[rbenv] or https://github.com/rvm/rvm[RVM] can be used to install compatible Ruby versions. Similarly, https://github.com/nvm-sh/nvm[nvm] can be used to manage node and npm on your system.
 
-Before installing PostgreSQL, check first that you don't already have it installed (by running `rpm -q postgresql`).
+=== Fedora
 
-You will want to make sure that you have postgresql-devel installed so the database gems can install properly.
-Also, you would also need gcc, ruby-devel, libxml-devel, libxslt-devel and libvirt-devel packages.
+Fedora ships with a too new Ruby. It's recommended to install https://github.com/rbenv/rbenv[rbenv] to install a compatible version:
+
+[source, bash]
+....
+sudo dnf install rbenv ruby-build-rbenv nodejs
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+source ~/.bashrc
+rbenv install 2.7.6
+....
+
+https://github.com/rvm/rvm[RVM] can also be used.
+
+=== Enterprise Linux 8
+
+By default EL8 ships with older versions, but newer modules are available:
+
+[source, bash]
+....
+dnf module enable ruby:2.7
+dnf module enable postgresql:12
+dnf module enable nodejs:14
+....
+
+Now install the development packages:
 
-For Fedora, CentOS or RHEL, you can issue the following commands to install all required packages:
 [source, bash]
 ....
 dnf groupinstall "Development Tools" "Development Libraries"
-dnf -y install gcc-c++ git ruby ruby-devel rubygems \
+dnf -y install gcc-c++ git ruby ruby-devel rubygems-bundler \
     libvirt-devel postgresql-devel openssl-devel \
     libxml2-devel libxslt-devel zlib-devel \
-    readline-devel systemd-devel tar libcurl-devel
+    readline-devel systemd-devel tar libcurl-devel nodejs
+....
+
+=== PostgreSQL
+
+Use the following steps to set up PostgreSQL on RPM based distributions:
+
+[source, bash]
+....
+sudo dnf install postgresql
+sudo postgresql-setup --initdb
+sudo -u postgres createuser --createdb $USER
```

Comment:
Do you need to be in a specific directory to run this?

Reply:
No, that shouldn't matter. I have seen warnings when the postgres user can't chdir to `$PWD`, but that's only a warning.